### ISSUE TITLE
How can

### ISSUE THREAD

**Issue by etale-cohomology** [perigoso/keyswitch-kicad-library#61]
Is it possible to use this in Kicad without symbols?
I've loaded the footprints into Kicad 6, but I don't know what to do when there are no symbols.

**Comment by perigoso** [comment]
Yes, use a generic symbol, like a simple switch symbol, and on the footprint association tie them to these footprints

**Comment by etale-cohomology** [comment]
Thanks. I see that [marbastlib](https://github.com/ebastler/marbastlib) has symbols for Choc switches.
Is it possible to use those symbols as-is, without change, for the **Choc Mini** footprints in this repo?
